content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md

```diff
@@ -0,0 +1,245 @@
+- Instalação de drivers de dispositivo específicos ou serviços Windows.
+- Consolidação de tarefas administrativas e políticas de segurança. 
+Isso reduz o grau de privilégios necessários para os nós do Windows.
+
+
+## {{% heading "prerequisites" %}}
+
+<!-- change this when graduating to stable -->
+
+Este guia de tarefas é específica para o Kubernetes v{{< skew currentVersion >}}.
+Se você não está executando o Kubernetes v{{< skew currentVersion >}}, 
+verifique a documentação para essa versão do Kubernetes.
+
+No Kubernetes {{< skew currentVersion >}}, O recurso de contêiner do HostProcess 
+é ativado por padrão. 
+O kubelet irá comunicar-se com o contêiner diretamente passando a flag HostProcess 
+via `CRI`. Você pode usar a versão mais recente do containerd (v1.6+) para executar 
+contêineres HostProcess.
+[Como instalar o containerd.](/docs/setup/production-environment/container-runtimes/#containerd)
+
+Para *desabilitar* contêineres HostProcess você precisa passar a seguinte flag 
+de portal de funcionalidade para o **kubelet** e **kube-apiserver**:
+
+```powershell
+--feature-gates=WindowsHostProcessContainers=false
+```
+
+Veja a documentação [Portal de Funcionalidade](/docs/reference/command-line-tools-reference/feature-gates/#overview)
+para mais detalhes.
+
+
+
+## Limitações
+
+Essas limitações são relevantes para o Kubernetes v{{< skew currentVersion >}}:
+
+- Contêineres HostProcess requerem containerd 1.6 ou mais recente
+  {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}.
+- Pods HostProcess só podem conter contêineres HostProcess. Esta é uma limitação atual
+  do sistema operacional Windows; Contêineres do Windows sem privilégios não podem 
+  compartilhar um vNIC com o IP do host no namespace.
+- Contêineres HostProcess executam como um processo do host e não tem nenhum grau de
+  isolamento, além das restrições de recursos impostas à conta de usuário do HostProcess. 
+  Nenhum sistema de arquivos ou isolação do Hyper-V são suportados para contêineres 
+  HostProcess.
+- `Volume mounts` são suportados e são montados dentro do volume do contêiner. Veja
+  [Volume Mounts](#volume-mounts)
+- Um conjunto limitado de contas de usuário do host está disponível para os contêineres 
+  HostProcess por padrão.
+  Veja [Escolhendo uma conta de usuário](#choosing-a-user-account).
+- Limites de recursos (disco, memória, número de cpus) são suportados da mesma maneira 
+  que os processos no host.
+- Ambos, `Named pipe mounts` e `Unix domain sockets` **não** são suportados e devem
+  ser acessados através de seu caminho no host (ex. \\\\.\\pipe\\\*)
+
+## Pod HostProcess requisitos de configuração
+
+Habilitar um Pod do Windows HostProcess requer a definição das configurações corretas 
+de segurança na configuração do pod. Das políticas definidas no 
+[Padrões de segurança de pod](/docs/concepts/security/pod-security-standards)
+Pods HostProcess são desabilitados pela linha de base e políticas restritas. 
```

Comment:
While your translation for _baseline_ and _restricted_ are very good, I think we should keep the original in English, as they refer to specific configurations.

Suggestion: "Entre as políticas definidas nos padrões de segurança de Pod, os Pods HostProcess não são permitidos nas políticas _baseline_ e _restricted_".